Visibility updates v1.21 (#2131)
temporalio/documentation: docs-src/references/configuration.md

## persistence

The `persistence` section holds configuration for the data store/persistence layer.
The following example shows a minimal specification for a password-secured Cluster using Cassandra.

```yaml
persistence:
  defaultStore: default

>   visibilityStore: cass-visibility # This is the primary Visibility store.
>   secondaryVisibilityStore: es-visibility # Secondary Visibility store added to enable Dual Visibility.

Reviewer: Optional changes:
```suggestion
  visibilityStore: cass-visibility # The primary Visibility store.
  secondaryVisibilityStore: es-visibility # A secondary Visibility store added to enable Dual Visibility.
```

```yaml
persistence:
  defaultStore: default
  visibilityStore: cass-visibility

>   secondaryVisibilityStore: es-visibility

Reviewer 2: Why add secondary visibility config here?

Author: Because this is an example of persistence store config that can be used as a reference for the keys that can be set here. I'll add comments here to explain this better.
